src/Microsoft.ML.DataView/VBuffer.cs

```diff
@@ -77,16 +117,27 @@ public VBuffer(int length, T[] values, int[] indices = null)
         }
 
         /// <summary>
-        /// Construct a possibly sparse representation.
+        /// Construct a possibly sparse vector representation.
         /// </summary>
+        /// <param name="length">The length of the constructed buffer.</param>
+        /// <param name="count">The count of explicit entries. This must be between 0 and <paramref name="count"/>, both
```

Comment:
`This must be between 0 and <paramref name="count"/>`

=>

`This must be between 0 and <paramref name="length"/>`